docs/rst/tutorials/cloud/edge_edge_repeater_cloud/edge_edge_repeater_cloud.rst

```diff
@@ -0,0 +1,346 @@
+
+Then source the ``turtlesim_move_ws`` workspace:
+
+.. code-block:: bash
+
+    source turtlesim_move_ws/install/setup.bash
+
+And finally, run the application:
+
+.. tabs::
+
+    .. tab:: WAN
+
+        .. code-block:: bash
+
+            ros2 run turtlesim turtlesim_square_move
+
+    .. tab:: LAN
+
+        .. code-block:: bash
+
+            ROS_DOMAIN_ID=1 ros2 run turtlesim turtlesim_square_move
+
+        .. note::
+
+            As stated :ref:`here <warning_lan>`, change the ROS 2 Domain Id if running the edge and cloud applications on the same LAN.
+
+
+The important points to note in this application are the following:
+
+*   The control application sends the movement commands to the ``turtlesim_node`` through a publisher in the `geometry_msgs/msg/Twist <https://docs.ros2.org/foxy/api/geometry_msgs/msg/Twist.html>`_ topic.
+    This topic expresses the velocity at which the turtle has to move divided into linear velocity (``linear``) and angular velocity (``angular``).
+
+*   The application knows the position of the turtle on the map at any moment and whether it is stopped or moving by subscribing to the `turtlesim/msg/pose <https://docs.ros2.org/foxy/api/turtlesim/msg/Pose.html>`_ topic.
+    This topic provides information about the turtle's coordinates (``x`` and ``y``) and the turtle's rotation (``theta``). We can also know its linear and angular velocity (``linear_velocity`` and ``angular_velocity``).
+
+
+Running DDS Router Edge 2
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the Repeater server is the same for both edges, the configuration of the DDS Router Edge 2 is similar same as
+:ref:`the one for the DDS Router Edge 1 <tutorials_cloud_edge_edge_repeater_cloud_edge_edge_repeater_cloud_running_router_1>`.
+In this example both edges use different ports to communicate with the Repeater, simulating 2 different networks available in the Cloud.
+However this is not needed, and only one address could be used.
+The following snippet shows a configuration file (changing Domain for LAN scenarios):
+
+.. literalinclude:: /resources/tutorials/cloud/edge_edge_repeater_cloud/dds_router_edge_2.yaml
+    :language: yaml
+
+The following figure summarizes the deployment on the edge 2.
+
+.. todo::
+
+    add new image
+
+.. figure:: /rst/figures/tutorials/cloud/edge_2_repeater_deployment.png
+   :align: center
+
+To finish this step, run the DDS Router with the configuration file created as an argument.
+
+.. code-block::
+
+    ddsrouter -c <path/to/file>/ddsrouter_edge_2.yaml
+
+Deployment of Cloud Repeater
+----------------------------
+
+In order to communicate both edges, we use a DDS Router configuring as a *Repeater* that forward the messages from one edge to the other.
+This machine should be accessible from the Internet.
+In case this device is working under a NAT, check :ref:`previous turotial <tutorials_cloud_wan_edge_cloud_wan_edge_cloud_configure_transversal_nat>`
```

Comment:
```suggestion
In case this device is working under a NAT, check :ref:`previous tutorial <tutorials_cloud_wan_edge_cloud_wan_edge_cloud_configure_transversal_nat>`
```